dnsimple/resource_dnsimple_lets_encrypt_certificate_test.go

```diff
@@ -3,11 +3,12 @@ package dnsimple
 import (
 	"context"
 	"fmt"
-	"github.com/dnsimple/dnsimple-go/dnsimple"
 	"os"
 	"strconv"
 	"testing"
 
+	"github.com/dnsimple/dnsimple-go/dnsimple"
```

Comment:
Isn't this line redundant? It seems you have the same include in the next line.

Reply:
I would say the line below is redundant, as we know underscore imports are conventionally used to get their side effects. Whereas here we also rely on the actual package itself.

Thanks for bringing it up I had missed this anomaly entirely.

Reply:
Changed in [8e3122d](https://github.com/dnsimple/terraform-provider-dnsimple/pull/62/commits/8e3122d67d214633c0909b5a74710b821d5d5bde)